bindings/gumjs/gumdukcore.c

```diff
@@ -2753,7 +2755,9 @@ gum_duk_native_function_invoke (GumDukNativeFunction * self,
 
   if (gum_exceptor_catch (core->exceptor, &exceptor_scope))
   {
+    gum_interceptor_ignore_current_thread (core->interceptor);
     _gum_duk_throw_native (ctx, &exceptor_scope.exception, core);
+    gum_interceptor_unignore_current_thread (core->interceptor);
```

Comment:
This `unignore` call is unreadable code, as Duktape's exception handling unwinds the stack.